mlir/lib/ExecutionEngine/ROCm/CMakeLists.txt

```diff
@@ -94,9 +97,11 @@ add_llvm_library(LLVMROCmBackendUtils
   )
 
 llvm_update_compile_flags(LLVMROCmBackendUtils)
-target_include_directories(LLVMROCmBackendUtils
-  PRIVATE
-  "${HIP_PATH}/../include"
-  "${HIP_PATH}/include"
-)
+if ( USE_ROCM_4_4_OR_OLDER )
+  target_include_directories(LLVMROCmBackendUtils
+    PRIVATE
+    "${HIP_PATH}/../include"
+    "${HIP_PATH}/include"
+  )
+endif()
```

Comment:
How confidence we are in this feature hitting ROCm 4.4? I thought this is going to be ROCm 4.5. Could you double check if this is already on release schedule of the rocminfo team?

Redundant space around `USE_ROCM_4_4_OR_OLDER`.

Could you test locally such that, the default populated fat library, when link with MIOpen (In OCL backend), does not produce any linker errors

Reply:
Yes. This feature will be in ROCm4.5. The flag says if we use ROCm4.4 or any version before 4.4, we need HIP.

I've tested MIOpen with OpenCL backend and confirmed that all linker errors were gone.

Reply:
> I've tested MIOpen with OpenCL backend and confirmed that all linker errors were gone.

Thank you!